src/Mod/TechDraw/Gui/QGIViewAnnotation.cpp

```diff
@@ -189,4 +191,54 @@ void QGIViewAnnotation::rotateView(void)
     m_textItem->setRotation(-rot);
 }
 
+void QGIViewAnnotation::mouseDoubleClickEvent(QGraphicsSceneMouseEvent* event)
+{
+    Q_UNUSED(event);
+
+    TechDraw::DrawViewAnnotation *annotation = dynamic_cast<TechDraw::DrawViewAnnotation *>(getViewObject());
+    if (annotation == nullptr) {
+        return;
+    }
+
+    const std::vector<std::string> &values = annotation->Text.getValues();
+    QString text;
+    if (values.size() > 0) {
+        text = QString::fromUtf8(Base::Tools::escapedUnicodeToUtf8(values[0]).c_str());
+
+        for (unsigned int i = 1; i < values.size(); ++i) {
+            text += QChar::fromLatin1('\n');
+            text += QString::fromUtf8(Base::Tools::escapedUnicodeToUtf8(values[i]).c_str());
+        }
+    }
+
+    QDialog dialog(0);
+    dialog.setWindowTitle(tr("Text"));
+
+    Gui::PropertyListEditor editor(&dialog);
```

Comment:
This does not compile for me. I get:

`
2>QGIViewAnnotation.obj : error LNK2019: unresolved external symbol "public: __cdecl Gui::PropertyListEditor::PropertyListEditor(class QWidget *)" (??0PropertyListEditor@Gui@@QEAA@PEAVQWidget@@@Z) referenced in function "protected: virtual void __cdecl TechDrawGui::QGIViewAnnotation::mouseDoubleClickEvent(class QGraphicsSceneMouseEvent *)" (?mouseDoubleClickEvent@QGIViewAnnotation@TechDrawGui@@MEAAXPEAVQGraphicsSceneMouseEvent@@@Z)
`

Reply:
I found now out that to fix the compilation, this change is necessary (you can take the whole block as being a Diff file):

```
 src/Gui/Widgets.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Gui/Widgets.h b/src/Gui/Widgets.h
index 3f868697cf..86baff6a51 100644
--- a/src/Gui/Widgets.h
+++ b/src/Gui/Widgets.h
@@ -373,7 +373,7 @@ private:
 
 // ----------------------------------------------------------------------
 
-class PropertyListEditor : public QPlainTextEdit
+class GuiExport PropertyListEditor : public QPlainTextEdit
 {
     Q_OBJECT
 

```

Reply:
Nice catch Uwe, thank You. Unfortunately I do not have access to a Windows machine with Visual Studio installed and Linux linker handles DLL exports/imports differently.